java/compaction-status-store/src/main/java/sleeper/compaction/dynamodb/job/DynamoDBCompactionJobStatusStore.java

```diff
@@ -15,8 +15,32 @@
  */
 package sleeper.compaction.dynamodb.job;
 
```

Comment:
I think this would be better in a package like `sleeper.compaction.status.job` - I think status is definitely needed in the package name, and I don't think DynamoDB is as at the moment we're not planning on offering anything other than DynamoDB for the status store.

Reply:
Done